src/main/resources/entity-types/openaccess/drv_openaccess_party.json5

```
@@ -0,0 +1,57 @@
+{
+  id: '0f8f6d6f-504f-4aa9-9d98-9779aaaa907b',
+  name: 'drv_openaccess_party',
+  private: false,
+  fromClause: 'src_oa_party AS party\nLEFT JOIN src_oa_alternate_email_address ON alternate_email_address.aea_owner_fk=party.p_id\nLEFT JOIN src_oa_refdata_value AS p_institution_level_1 ON p_institution_level_1.rdv_id=party.p_institution_level_1_fk',
```

It doesn't look like `src_oa_alternate_email_address` or `src_oa_refdata_value` are actually used, so this could just be 
```suggestion
  fromClause: 'src_oa_party AS party',
```

---

they will be used for more columns, I just wanted to start testing with a few columns only